components/logins/ios/Logins/LoginRecord.swift

```diff
@@ -124,8 +123,8 @@ open class LoginRecord {
 
             username: dict["username"] as? String ?? "",
```

Comment:
Should we do the same for `username`? It is technically optional, but I'm not sure if there's a distinction with how an empty `username` gets stored to the DB.

Reply:
Ah, nice catch.

Reply:
For `username`, `username_field`, and `password_field`, it looks like empty strings are treated the same as nil. Which is... kind of dodgy. But yeah, it doesn't make a difference. I'll change them to be nil to reflect what actually comes over from rust though.

Reply:
(Actually, I don't think it's dodgy, I prefer empty strings to NULL in the DB, even if it's just because they work more predictably in comparisons. Anyway, this shouldn't have been exposed, that way, I'll make the change.)